Jenkinsfile

```diff
@@ -24,40 +24,11 @@ node ('rocmtest')
     deleteDir( )
```

Comment:
In Jenkins, it takes between 10 to 15 minutes to checkout the repo, thats longer than the build which is about 7 minutes.